README.md

```diff
@@ -10,6 +10,24 @@
 [![Gitter](https://badges.gitter.im/jenkinsci/configuration-as-code-plugin.svg)](https://gitter.im/jenkinsci/configuration-as-code-plugin)
 
 <img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
+## Documentation
```

Comment:
I think we could remove this heading now it’s at the top?

Reply:
done